src/hw-cbmc/set_inputs.cpp

```diff
@@ -71,6 +71,6 @@ void add_set_inputs(
   symbol.value=block;
 
   // hide and inline
-  symbol.type.set(ID_C_hide, true);
+  // symbol.type.set(ID_C_hide, true);
```

Comment:
This was removed in diffblue/cbmc@4e5bf6ea2bd03, and we might want to replace this code rather than just comment it out?